### PR TITLE
#159951820 Users can login via Google and Facebook

### PR DESCRIPTION
**Pull request by lytes20**
#### What does this PR do?
- Enable facebook and google login
#### Description of Task to be completed?
- On the Login route, users should also be able to Login via their favorite social media platform.
#### How should this be manually tested?
- By running the app on [Heroku](https://authors-haven-front.herokuapp.com/) and trying to log in with google or facebook
#### Any background context you want to provide?
- The app is now registered on the google and facebook developer consoles with the heroku url. For testing purposes please test from the heroku hosted branch [here](https://authors-haven-front.herokuapp.com/)
#### What are the relevant pivotal tracker stories?
[#159951820](https://www.pivotaltracker.com/story/show/159951820)
#### Screenshots
screen shot 2018-09-10 at 07 56 58

#### Questions:

### CODE REVIEW

File: src/tests/components/Google.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { GoogleLoginComponent } from '../../components/SocialLogin/GoogleLogin';
+
+describe('GoogleComponent', () => {
+  it('renders one componet', () => {
+    const componet = shallow(<GoogleLoginComponent />);
+    expect(componet).toHaveLength(1);
+  });
+});
```
Comment (geofrocker) [inline src/tests/components/Google.test.js]: missing EOF

File: src/tests/components/Facebook.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    // mockAxios.post.mockImplementation(() => Promise.resolve({
+    //   data: {
+    //     user: {
+    //       token: 'shbd',
+    //       username: 'testUsername',
+    //       email: 'email@email.com',
+    //     },
+    //   },
+    // }));
```
Comment (abulojoshua1) [inline src/tests/components/Facebook.test.js]: @lytes20, please take out the commented out code if it is not necessary.

File: src/components/SocialLogin/FacebookLogin.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+export class FacebookLoginComponent extends Component {
+  responseFacebook = fbResponse => {
+    const data = { user: { fbauth_token: fbResponse.accessToken } };
+    axios
+      .post('https://authors-haven-tabs.herokuapp.com/api/users/fbauth/', data)
+      .then(response => {
+        this.handleResponse(response, fbResponse);
+      })
+      .catch();
```
Comment (abulojoshua1) [inline src/components/SocialLogin/FacebookLogin.js]: @lytes20, why should you use an empty `.catch()` function?

**Review by abulojoshua1 — DISMISSED** [review]

**Comment by geofrocker** [comment]
Social login still returns the earlier mentioned error

**Comment by ssewilliam** [comment]
@lytes20, @AGMETEOR, have a look at this functionality in the back end as well.
Here I tried posting the data to the API route using postman and I still got the same error.

screen shot 2018-09-11 at 08 24 48

**Pull request merged**